src/graph/dot.ts

```diff
@@ -80,17 +82,24 @@ function nodeLabel(node) {
     node.name
   }</FONT><BR/>${kindLabel}</TD>
       </TR>
-      ${objectValues(node.fields, nodeField)}
+      ${objectValues(node.fields, nodeField, displayOptions)}
       ${possibleTypes(node)}
       ${derivedTypes(node)}
     </TABLE>>
   `;
 }
 
-function nodeField(field) {
+function canDisplayRow(type, displayOptions) {
+  if(type.kind === 'SCALAR') {
```

Comment:
@soneymathew Scalar doesn't include enums. Do you want to skip only scalars or any leaf fields (Scalars + Enums)?